### PR TITLE
BETA: Register potato.is-a.dev

### PR DESCRIPTION
**Pull request by RaafatTurki**
Added `potato.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Review by YouFoundAlpha — APPROVED** [review]
Looks good
Now you'll just have to wait for a maintainer to merge it

**Review by DEV-DIBSTER — APPROVED** [review]

**Pull request merged**
